### PR TITLE
improve debug logging for draining test

### PR DESCRIPTION
**Pull request by xh3b4sd**
Fixes https://github.com/giantswarm/giantswarm/issues/3001.

### CODE REVIEW

**Comment by xh3b4sd** [comment]
Fishing for LGTM.

File: integration/test/draining/draining_test.go [hunk 1]
```diff
@@ -212,15 +264,21 @@ func Test_Draining(t *testing.T) {
...
+		newLogger.Log("level", "debug", "message", "verified e2e-app availability 10 more seconds")
 	}
 
+	close(done)
+
 	{
 		newLogger.Log("level", "debug", "message", "validating test data")
 
-		acceptable := float64(20)
+		acceptable := float64(0)
```
Comment (MarcelMue) [inline integration/test/draining/draining_test.go]: Does this succeed reliably right now?

Comment (xh3b4sd) [reply integration/test/draining/draining_test.go]: I talked about this in standup. Yes it does.

Comment (xh3b4sd) [reply integration/test/draining/draining_test.go]: If it degrades we can easily bump the threshold again.

**Pull request merged**